server/src/index.js

```diff
@@ -1,67 +1,72 @@
 const express = require('express')
-const fs = require('fs')
 const path = require('path')
+const http = require('http')
+const fs = require('fs')
 
 const { ApolloVoyagerServer } = require('@aerogear/apollo-voyager-server')
+const { KeycloakSecurityService } = require('@aerogear/apollo-voyager-keycloak')
+const metrics = require('@aerogear/apollo-voyager-metrics')
+const auditLogger = require('@aerogear/apollo-voyager-audit')
 
+const config = require('./config/config')
 const connect = require('./db')
-const schema = require('./schema')
-const http = require('http');
+const { typeDefs, resolvers } = require('./schema')
 
-const dbOptions = {
-  database: process.env.DB_NAME || 'users',
-  user: process.env.DB_USERNAME || 'postgresql',
-  password: process.env.DB_PASSWORD || 'postgres',
-  host: process.env.DB_HOSTNAME || '127.0.0.1',
-  port: process.env.DB_PORT || '5432'
-}
+let keycloakService = null
 
-const PORT = 4000
+// if a keycloak config is present we create
+// a keycloak service which will be passed into
+// ApolloVoyagerServer
+if (config.keycloakConfig) {
+  keycloakService = new KeycloakSecurityService(config.keycloakConfig)
+}
 
 async function start() {
+  
   const app = express()
+  const httpServer = http.createServer(app)
 
-  app.get("/health", (req, res) => {
-    return res.json({});
-  });
+  metrics.applyMetricsMiddlewares
```

Comment:
what endpoint does the metrics middleware use? should it be configurable?

Reply:
Good catch, i'm actually not even calling the function here. Just updated that. It takes an a second options argument e.g. `{ path: '/whatever' }` but by default it will do `/metrics`. If you want I can add it in explicitly though!

Reply:
+1 on adding it explicitly. Make it more clear to users. But I am getting an error when I request that endpoint. Does it working for you?